tests/installation/licensing/verify_license_translations.pm

```diff
@@ -55,7 +55,7 @@ sub run {
         $license_agreement->select_language($translation->{language});
         $license_agreement_info = $license_agreement->collect_current_license_agreement_info();
         if ($license_agreement_info->{text} !~ /$translation->{text}/) {
-            $errors .= "EULA content for the language: '$translation->{language}' didn't validate. Please, see autoints-log for the detailed content of EULA\n";
+            record_soft_failure("EULA content for the language: '$translation->{language}' didn't validate. Please, see autoints-log for the detailed content of EULA\n");
```

Comment:
i wonder if you should check for BETA in this case as IIUC what you want is to prevent false negatives in this stage of the project. Do we expect those translations to be ready in BETA? Or we are not sure?

Reply:
I raised this question in our daily and got assured, that nothing will be overseen in the reviews.

Reply:
So the problem is that there is no way to automate the logic or what? As a QA I would say, do not trust anyone, an assurance wont do it. And it is not exactly justified. But its fine i guess if you want to oversee it for now. As minimum requirement i would add a comment in the softfail for the reviewer, for instance something like "This should not softfail in GA" or "Do not oversee this after BETA" or something. WDYT?